easybuild/easyconfigs/h/HERRO/HERRO-0.1.0_20240808-foss-2023a.eb

```diff
@@ -0,0 +1,382 @@
+easyblock = 'Cargo'
+
+name = 'HERRO'
+version = '0.1.0_20240808'
+local_commit = 'deccc46'
+
+homepage = 'https://github.com/lbcb-sci/herro'
+description = """
+HERRO is a highly-accurate, haplotype-aware, deep-learning tool for error correction of Nanopore R10.4.1 or
+R9.4.1 reads (read length of >= 10 kbps is recommended).
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/lbcb-sci/herro/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+
+builddependencies = [
+    ('Rust', '1.75.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('bzip2', '1.0.8'),
+    ('libffi', '3.4.4'),
+    ('libnsl', '2.0.1'),
+    ('SQLite', '3.42.0'),
+    ('util-linux', '2.39'),
+    ('zlib', '1.2.13'),
+    ('ncurses', '6.4'),
+    ('libreadline', '8.2'),
+    ('SeqKit', '2.3.1', '', SYSTEM),
+    ('Tk', '8.6.13'),
+    ('XZ', '5.4.2'),
+    ('matplotlib', '3.7.2'),
+    ('edlib', '1.3.9'),
+    ('h5py', '3.9.0'),
+    ('duplex-tools', '0.3.3'),
+    ('PyTorch', '2.1.2'),
+    ('Pillow', '10.0.0'),
+    ('zstd', '1.5.5'),
+    ('parasail', '2.6.2'),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'download_dep_fail': True,
+    'use_pip': True,
+    'sanity_pip_check': True,
+    'preinstallopts': '',
+    'installopts': '',
+}
+
+exts_list = [
+    ('zstandard', '0.22.0', {
+        'checksums': ['8226a33c542bcb54cd6bd0a366067b610b41713b64c9abec1bc4533d69f51e70'],
+    }),
+    ('Porechop', '20240119', {
+        'modulename': 'porechop',
+        'source_urls': ['https://github.com/dehui333/Porechop/archive/'],
+        'sources': [{'download_filename': 'd2e77c6.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'checksums': ['6e5ff3a780fc2855b0101b4a6102437d9a0fc201e40ffabc44c0c67d7c9ad621'],
+    }),
+]
+
+# use tch version 0.14.0 - compatible with PyTorch 2.1
+local_torch_opts = "sed -i 's/tch = \"0.13.0\"/tch = \"0.14.0\"/' Cargo.toml && "
+local_torch_opts += 'export LIBTORCH_BYPASS_VERSION_CHECK=1 && export LIBTORCH_USE_PYTORCH=1 && '
+local_torch_opts += 'export LIBTORCH=$EBROOTPYTORCH/lib && RUSTFLAGS="-Ctarget-cpu=native"'
+prebuildopts = pretestopts = preinstallopts = local_torch_opts
+
+# copy scripts to /bin to be easier to run scripts for users
+postinstallcmds = [
+    'cp -a %(start_dir)sscripts/* %(installdir)s/bin',
```

Comment:
```suggestion
    'cp -a %(start_dir)s/scripts/* %(installdir)s/bin',

Reply:
I had it like this: 'cp -a %(start_dir)s/scripts/* %(installdir)s/bin' but it make 2x // before scripts